cmd/spiffe-aws-assume-role/main.go

```diff
@@ -2,13 +2,19 @@ package main
 
 import (
 	"fmt"
+	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
 	"os"
 
 	"github.com/square/spiffe-aws-assume-role/cmd/spiffe-aws-assume-role/cli"
 )
 
 func main() {
-	if err := cli.Run(os.Args[1:]); err != nil {
+	context := cli.CliContext{
+		JWTSourceProvider: credentials.StandardJWTSourceProvider,
+		STSProvider:       credentials.StandardSTSProvider,
+	}
+
+	if err := cli.Run(os.Args[1:], &context); err != nil {
 		_, _ = fmt.Fprintf(os.Stderr, "error: %v\n", err)
 		os.Exit(1)
 	}
```

Comment:
In our quest for testability we've introduced some inversion of control. Specifically, the JWT source and STS client are "injected" via factory methods on our context.